tailwind.config.js

```diff
@@ -1,8 +1,20 @@
 module.exports = {
-  mode: "jit",
+  mode: 'jit',
   theme: {
     container: {
       center: true,
     },
+    extend: {
+      fontFamily: {
+        display: ['EB Garamond', 'Hind'],
+      },
```

Comment:
We need to register the fonts separate of each other so we can specifically call for garamond or hind.

I'd do something like this

```
fontFamily: {
  display: '['EB Garamond', 'serif'],
  body: ['Hind', 'sans-serif'],
 }
```